src/xdocs/config_whitespace.xml

```diff
@@ -259,11 +259,19 @@ for (Iterator foo = very.long.line.iterator();
                 <td>6.18</td>
             </tr>
           <tr>
+             <td>treatStaticImportAsImport</td>
+             <td>Allows no empty line between import and static import.</td>
+             <td><a href="property_types.html#boolean">Boolean</a></td>
+             <td>false</td>
+             <td>8.1</td>
```

Comment:
please make it 8.3.

8.2 will be in few days, but there will be some changes and I will request testing report.